config.toml

```diff
@@ -19,7 +19,19 @@ bestqualityfirst = true
 # EXPERIMENTAL enable temporal layer change is currently an experimental feature,
 # enable only for testing.
 enabletemporallayer = false
-
+# Sets the audio level volume threshold.
+# Values from [0-127] where 0 is the loudest.
+# Audio levels are read from rtp extension header according to:
+# https://tools.ietf.org/html/rfc6464
+audiolevelthreshold = 40
+# Sets the interval in which the SFU will emit the audio level
+# events in [ms]
+audiolevelinterval=1000
```

Comment:
does this mean a client will only receive audio updates every second? is it possible to instead trigger based on a new active speaker?

Reply:
It will only send the message if the speaker(s) has changed, so you could set it to quite a short interval. I guess the comment should now say `in which the SFU will **check** the audio level`.

Reply:
the problem is to identify when is there an active speaker and not a noise, the audiolevel will not send an event if there is no changes in the speakers, but it will wait the interval to check the loudest peer. so fi Peer A is speaking for 1 minute there will be only one speaker event in the DC API